app/views/projects/_table.html.erb

```diff
@@ -14,7 +14,9 @@
   </thead>
 
   <tbody>
+
     <% projects.each do |project| %>
+    <% cache [project.id, @result] do %>
```

Comment:
You don't want to do that.  The key will only have the project.id, so when the project is updated it'll show the OLD value.  Also, why use "@result" here?  Won't it just be nil?